index.js

```diff
@@ -6,9 +6,7 @@ module.exports = function () {
     return function () {};
   }
   var active = document.activeElement;
-  var ranges = Array.apply(Array, {
-    length: selection.rangeCount
-  }).map(function(range, index) {
+  var ranges = Array(selection.rangeCount).map(function(range, index) {
```

Comment:
That still won't work. You cannot `map` through empty array, no matter what length it has. You can do `while` or `for`, but not `.map()`